### ISSUE TITLE
[Question] Refresh thumbnail every time a tile's URL is visited

### ISSUE THREAD

**Issue by iG8R** [fastaddons/GroupSpeedDial#266]
Hi.
Is there any way the tile's thumbnail would be refreshed each time I click on this tile and visit corresponding URL?

**Comment by Juraj-Masiar** [comment]
Hello,
I'm sorry for the delay.
This was considered long ago but there is many issues due to which it was not implemented. Currently when the page thumbnail is created the page is stripped of scrollbars, animations are removed, and the page is zoomed in/out (so that the elements of the page are bigger and more visible in the small thumbnail).
**Example - left is default, right is without applied zoom:**
![image](https://github.com/fastaddons/GroupSpeedDial/assets/6358971/df3ea7cc-cc14-489c-bb16-a029c7368199)

Additionally, taking screenshot consumes considerable amount of CPU which could further slow down page load (especially if you have high definition screen). Plus several IO write operation to the drive (not great for SSD).

That's why there is "Auto-reload" feature. Too bad it's not invisible in Chromium browsers, only in Firefox.

**Comment by Juraj-Masiar** [comment]
Interesting, have you noticed any issues while using it in the other extension?
One this that comes to mind is the timing - average page time load is about 3 seconds and many people will start scrolling the page before it's fully loaded. That would then result into "scrolled page thumbnail" - have you seen that in the other extension?

**Comment by iG8R** [comment]
No, I didn't notice any, even though I've changed the delay time before taking a screenshot to 4 sec.

**Comment by iG8R** [comment]
Hello.
Is there any progress on this issue?

**Comment by iG8R** [comment]
Thanks a lot for such a detailed answer.
I'll be really looking forward to it and keeping my fingers crossed! :)
